### PR TITLE
Added device Samsung Galaxy Note 2 (N7100) support (again!)

### PR DESCRIPTION
**Pull request by JamesHealdUK**
Hopefully this checks out alright :+1:

### CODE REVIEW

File: aports/device/linux-samsung-n7100/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,110 @@
+# APKBUILD based on linux-vanilla aport. Changes:
+# - disabled module installation
+# - add !check !tracedeps
+# - package: just install zimage and kernel.release, because the kernel config
+#	does not generate modules or dtb files
+# - do not create -dev subpackage (makes no sense without module support)
+#
+# Kernel config based on: arch/arm/configs/lineageos_mako_defconfig
```
Comment (PabloCastellano) [inline aports/device/linux-samsung-n7100/APKBUILD]: Please adjust accordingly

File: aports/device/linux-samsung-n7100/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+# Changes:
+# - enable devtmpfs (needed for udev -> touch support in weston)
+
+_vendor=samsung
+_flavor=samsung-n7100
+_hash="5790cd7defee9cb6f9b392af4e3569e51230917a"
+_config="config-${_flavor}.armhf"
+
+pkgname=linux-${_flavor}
+pkgver=3.0.31
```
Comment (PabloCastellano) [inline aports/device/linux-samsung-n7100/APKBUILD]: Please adjust. Is it [3.0.64](https://github.com/bigbiff/android_kernel_samsung_smdk4412/blob/cm-10.1/Makefile#L1-L3), right?

File: aports/device/linux-samsung-n7100/APKBUILD (hunk 1 above)
Comment (PabloCastellano) [inline aports/device/linux-samsung-n7100/APKBUILD]: Also, comment briefly the (3) patches that you have applied

File: aports/device/device-samsung-n7100/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+pkgrel=6
+pkgdesc="Samsung Galaxy Note II (GSM)"
+url="https://github.com/postmarketOS"
+arch="noarch"
+license="MIT"
+depends="linux-samsung-n7100 mkbootimg"
+makedepends=""
+install=""
+subpackages=""
+source="deviceinfo 90-android.rules"
```
Comment (PabloCastellano) [inline aports/device/device-samsung-n7100/APKBUILD]: Did you forget to include `initfs-hook.sh` here?

Comment (JamesHealdUK) [inline aports/device/device-samsung-n7100/APKBUILD]: It seems to be working without, so Im not sure its needed. I've noticed the i9100 doesnt have one either.

**Comment by JamesHealdUK** [comment]
Thanks for the review @PabloCastellano ! I hope all the changes are suitable.

File: aports/device/device-samsung-n7100/APKBUILD (hunk 3 above)
Comment (drebrez) [inline aports/device/device-samsung-n7100/APKBUILD]: @JamesHealdUK if it works without then you should delete the `aports/device/device-samsung-n7100/initfs-hook.sh` file.

**Comment by herbsmn** [comment]
Can someone add a blurb on the wiki page that links to http://blog.forkwhiletrue.me/pages/midas-mainline/ ? there's already a blurb on the samsung-n5110 wiki about it. if anyone is interested in collaborating with the Replicant project to mainline linux or liberating the bootloader for the n7100, hit up Putti or forkbomb in the #replicant channel on freenode or shoot an email to our email list.

**Comment by ollieparanoid** [comment]
@herbsmn: it would be really nice if you could add such information to the wiki by yourself. Just like @MayeulC writes, you don't need an e-mail address to register. So it would probably not have taken you much longer than writing in these three GitHub issues. This time I've done it for you though, and I've also mentioned `#replicant` in the [mainlining guide](https://wiki.postmarketos.org/wiki/Mainlining_Guide#Mentor).

I hope that someone with one of these devices shows up for collaboration sooner or later. Replicant is an awesome project and it's awesome that it's looking into mainling as well, thanks for reaching out!

**Pull request merged**